main.go

```diff
@@ -0,0 +1,182 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+	"time"
+)
+
+type container struct {
+	ID string `json:"container_id"`
+}
+
+const teBaseURL = "test-env.scalr.com"
+
+var (
+	revizorBaseURL = getEnv("REVIZOR_URL")
+	revizorToken   = getEnv("REVIZOR_TOKEN")
+	scalrToken     = getEnv("SCALR_TOKEN")
+)
+
+func getEnv(key string) string {
+	value, present := os.LookupEnv(key)
+	if !present || len(value) == 0 {
+		log.Fatalf("No required environment variable: %s", key)
+		return ""
+	}
+	return value
+}
+
+func newRequest(method, path string, payload interface{}) *http.Request {
+	reqBody := bytes.NewBuffer(nil)
+	reqHeaders := make(http.Header)
+	reqHeaders.Set("Authorization", fmt.Sprintf("Token %s", revizorToken))
+	if payload != nil {
+		reqHeaders.Set("Content-Type", "application/json")
+		jsonEncoded, err := json.Marshal(payload)
+		if err != nil {
+			log.Fatal(err)
+		}
+		reqBody.Write(jsonEncoded)
+	}
+	req, err := http.NewRequest(method, revizorBaseURL+path, reqBody)
+	if err != nil {
+		log.Fatal(err)
+	}
+	for k, v := range reqHeaders {
+		req.Header[k] = v
+	}
+	return req
+}
+
+func doHealthCheck(containerID *string) error {
+	// TODO: use ping endpoint within stable profile
+	url := fmt.Sprintf("https://%s.%s/api/iacp/v3/environments", *containerID, teBaseURL)
+	req, err := http.NewRequest("GET", url, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", scalrToken))
+	req.Header.Set("Prefer", "profile=preview")
```

Comment:
Would be great to have in a const as well.

Reply:
We will change it after [https://scalr-labs.atlassian.net/browse/SCALRCORE-17546](SCALRCORE-17546).